srcpkgs/mdcat/template

```diff
@@ -11,8 +11,8 @@ maintainer="cinerea0 <cinerea0@protonmail.com>"
 license="MPL-2.0"
 homepage="https://codeberg.org/flausch/mdcat/"
 changelog="https://codeberg.org/flausch/mdcat/raw/branch/main/CHANGELOG.md"
-distfiles="https://codeberg.org/flausch/mdcat/archive/mdcat-${version}.tar.gz"
-checksum=1120c4f3b5b517075b6347dbfe76a2211a91837b4d0242c0b72cdbc19e6886dd
+distfiles="https://codeberg.org/flausch/mdcat/archive/${pkgname}-${version}.tar.gz"
```

Comment:
It's generally preferred that distfiles don't use `${pkgname}`. Is there a specific reason in this case?

Reply:
I thought it would solve the "NO VERSION found for mdcat" error I get when running `xbps-src update-check`. It did not.

Reply:
For that you'd need to write an `update` file. It'd what update-check uses when the upstream isn't one it know how to deal with like Github

Reply:
Other packages like `foot` also download their sources from Codeberg, have a similar source URL, and have no issue checking for updates without an `update` file.

Reply:
Hum, there seems to be a definition for codeberg https://github.com/void-linux/void-packages/blob/master/common/xbps-src/shutils/update_check.sh#L146

You might need to specify a variable manually to get it to work

Reply:
Looking at it a bit more, the codeberg support works on releases, but mdcat only has tags
I *think* setting the url to the tag page instead of the release page should work

Reply:
Nevermind, they do appear to use releases but have not published one for the latest tag